tests/network-tests/src/Api.ts

```diff
@@ -53,7 +54,7 @@ export class ApiFactory {
     treasuryAccountUri: string,
     sudoAccountUri: string
   ): Promise<ApiFactory> {
-    const debug = Debugger('api-factory')
+    const debug = Debugger('integration-tests:api-factory')
```

Comment:
Good suggestion.
I had considered to pass around a 'global' debugger instance at one point, but changed my mind because I wasn't aware of that feature.